packages/navstar/extra/dcos-navstar.service

```diff
@@ -21,4 +21,7 @@ ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-navstar
 ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-minuteman
 ExecStartPre=/usr/bin/mkdir -p /var/lib/dcos/navstar/mnesia
 ExecStartPre=/usr/bin/mkdir -p /var/lib/dcos/navstar/lashup
+ExecStartPre=/usr/bin/env modprobe dummy
+ExecStartPre=-/usr/bin/env ip link add minuteman type dummy
```

Comment:
The number of ExecStartPre's here and now the addition of a `-` (fails are ignored) on this line suggest that we need to turn all of this into a proper startup script with error handling.